From: Stephen Rothwell <sfr@canb.auug.org.au>
Subject: MAINTAINERS: add Mark Brown as a linux-next maintainer
Date: Wed, 22 Oct 2025 16:36:25 +1100

Mark has been kindly helping fill in when I have been unavailable over the
past several years.  He has has also put his hand up to take over
linux-next maintenance when I finally decide to stop (which may be some
time yet ;-) ).

Link: https://lkml.kernel.org/r/20251022163625.5df7542a@canb.auug.org.au
Signed-off-by: Stephen Rothwell <sfr@canb.auug.org.au>
Acked-by: Mark Brown <broonie@kernel.org>
Cc: Linus Torvalds <torvalds@linux-foundation.org>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 MAINTAINERS |    1 +
 1 file changed, 1 insertion(+)

--- a/MAINTAINERS~maintainers-add-mark-brown-as-a-maintainer
+++ a/MAINTAINERS
@@ -14393,6 +14393,7 @@ F:	tools/memory-model/
 
 LINUX-NEXT TREE
 M:	Stephen Rothwell <sfr@canb.auug.org.au>
+M:	Mark Brown <broonie@kernel.org>
 L:	linux-next@vger.kernel.org
 S:	Supported
 B:	mailto:linux-next@vger.kernel.org and the appropriate development tree
_
